pages/trackingprotection.py

```diff
@@ -4,6 +4,8 @@
 
 class TrackingProtectionPage(Page):
 
+    URL_TEMPLATE = 'https://mozilla.github.io/tracking-test/'
```

Comment:
Fair enough- it was on another page also, I'll put that on the cleanup list for future reference!